mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2868,6 +2872,36 @@
     <string name="protection_panel_etp_toggle_enabled_description">If something looks broken on this site, try turning off protections.</string>
     <!-- Description text for the Enhanced Tracking Protection toggle when tracking protection is disabled. -->
     <string name="protection_panel_etp_toggle_disabled_description">Protections are OFF. We suggest turning them back on.</string>
+    <!-- Text for displaying the total number of trackers blocked in the protection panel.
+        %d is a placeholder for the number of trackers blocked. -->
+    <string name="protection_panel_num_trackers_blocked">%d Trackers blocked</string>
```

Comment:
Please do like in `<string name="trackers_blocked_panel_total_num_trackers_blocked">Trackers blocked: %d</string>` since we don't yet support proper plurals